indico/modules/rb/controllers/backend/misc.py

```diff
@@ -27,15 +29,27 @@
 from indico.modules.rb.util import build_rooms_spritesheet
 from indico.util.caching import memoize_redis
 from indico.util.i18n import get_all_locales
+from indico.util.string import sanitize_html
 from indico.web.flask.util import send_file, url_for
 
 
 class RHConfig(RHRoomBookingBase):
     def _process(self):
+        url = legal_settings.get('tos_url')
+        if url:
+            tos_type = 'url'
```

Comment:
matter of personal opinion, but i'd probably just send `tos_url` and `tos_html`.

anyway, with a `tos_type` the case of not having tos at all should be covered (`'none'` or `null`) as well